aws-cloudformation-schema/aws-apigateway-authorizer.json

```diff
@@ -1,78 +1,48 @@
 {
   "typeName" : "AWS::ApiGateway::Authorizer",
-  "description" : "Represents an authorization layer for methods. If enabled on a method, API Gateway will activate the authorizer when a client calls the method.",
-  "sourceUrl" : "https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-apigateway.git",
+  "description" : "Resource Type definition for AWS::ApiGateway::Authorizer",
```

Comment:
Hmm, seems like we're losing useful data again.

Reply:
We are still discussing the details with AWS folks, so I suggest we ignore this for now. We'll start tracking breaking changes after their advice is fully implemented.